vscode-dotnet-runtime-library/src/Utils/IssueReporter.ts

```diff
@@ -7,7 +7,7 @@ import { IIssueContext } from './IIssueContext';
 // tslint:disable no-var-requires
 const packageJson = require('../../package.json');
 
-const issuesUrl = `https://github.com/dotnet/vscode-dotnet-runtime/issues`;
+const issuesUrl = `https://github.com/dotnet/vscode-dotnet-runtime/issues/new?assignees=&labels=&projects=&template=bug-report--vs-code-runtime-extension.md&title=`;

```

Comment:
Does this redirect to the bug report template? I think it should instead redirect to https://github.com/JL03-Yue/vscode-dotnet-runtime/issues/new/choose something like the /choose url

Reply:
I believe it redirect to the bug report template when we merged the templates as well. It redirects to the correct bug report template when using the forked repo

Reply:
I think we should redirect to the main choose template issue instead, though I can see why you might want to default to the bug template because the user has encountered a bug most likely if they got here. With that said, I think we should redirect users to the main issue template page instead as it allows them to report an issue using a different template if they want to make a feature request instead. (It also forces them to think about the issue before they submit it, so people do not just click submit issue without reading and never give us anything of value.)